packages/lab/stories/drawer/drawer.stories.tsx

```diff
@@ -0,0 +1,356 @@
+import { useState, MouseEvent } from "react";
+import {
+  Button,
+  FlowLayout,
+  StackLayout,
+  FlexItem,
+  FlexLayout,
+  FormField,
+  FormFieldLabel,
+  FormFieldHelperText,
+  Input,
+} from "@salt-ds/core";
+import { CloseIcon } from "@salt-ds/icons";
+import { Drawer, DRAWER_POSITIONS } from "@salt-ds/lab";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import "./drawer.stories.css";
+
+export default {
+  title: "Lab/Drawer",
+  component: Drawer,
+  argTypes: {
```

Comment:
Was there an issue relying on the default types?

Reply:
This was just for adding the storybook dropdown which isn't really necessary so I've removed it